vms/components/avax/camino_utxo_with_msig.go

```diff
@@ -0,0 +1,16 @@
+package avax
+
+import "github.com/ava-labs/avalanchego/vms/components/verify"
+
+type UTXOWithMSig struct {
+	UTXO       `serialize:"true"`
+	Alias      verify.State `serialize:"true" json:"alias"`
+	ExportedAt uint64       `serialize:"true" json:"exportedAt"`
+
+	// The destination chain's recipient of the funds. It will overwrite the `To` field of the ImportTx for this particular UTXO
+	Recipient [20]byte `serialize:"true" json:"recipent"`
```

Comment:
why not shortID?

Reply:
well, for example, from what I see, c-chain export tx also imports ids package and uses ShortID